recipe/meta.yaml

```diff
@@ -1,17 +1,26 @@
-{% set name = "polars" %}
+{% if polars_variant == "polars-lts-cpu" %}
+  {% set name = "polars-lts-cpu" %}
+{% elif polars_variant == "polars-u64-idx" %}
+  {% set name = "polars-u64-idx" %}
+{% else %}
+  {% set name = "polars" %}
+{% endif %}
 {% set version = "0.20.31" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
```

Comment:
One use 1 feedstock for essentially three different packages?

Reply:
Yes, I considered using multi-outputs with archspec and track features for the lts variant, however I am not aware how we would do so for the bigidx-variant. The build only differs by the used build-flags and rust-flags, so I think just building multiple packages is the easiest way. Note that setting the RUSTFLAGS also requires a complete recompilation, so I don't think this would really profit from using multi-outputs.